cookienotice/templates/head.tpl

```diff
@@ -0,0 +1,40 @@
+<!-- <link rel="stylesheet" type="text/css" href="/addon/cookienotice/css/cookienotice.css" /> -->
+<script>
+	window.addEventListener("load", function () {
```

Comment:
Please move the Javascript code in a new `cookienotice.js` file.

Reply:
done